llpc/context/llpcCompiler.cpp

```diff
@@ -1485,14 +1498,26 @@ Result Compiler::BuildComputePipeline(const ComputePipelineBuildInfo *pipelineIn
 // @param options : An array of compilation-option strings
 MetroHash::Hash Compiler::generateHashForCompileOptions(unsigned optionCount, const char *const *options) {
   // Options which needn't affect compilation results
-  static StringRef IgnoredOptions[] = {
-      cl::PipelineDumpDir.ArgStr, cl::EnablePipelineDump.ArgStr, cl::ShaderCacheFileDir.ArgStr,
-      cl::ShaderCacheMode.ArgStr, cl::EnableOuts.ArgStr,         cl::EnableErrs.ArgStr,
-      cl::LogFileDbgs.ArgStr,     cl::LogFileOuts.ArgStr,        cl::ExecutableName.ArgStr};
+  static StringRef IgnoredOptions[] = {cl::PipelineDumpDir.ArgStr,
+                                       cl::EnablePipelineDump.ArgStr,
+                                       cl::ShaderCacheFileDir.ArgStr,
+                                       cl::ShaderCacheMode.ArgStr,
+                                       cl::EnableOuts.ArgStr,
+                                       cl::EnableErrs.ArgStr,
+                                       cl::LogFileDbgs.ArgStr,
+                                       cl::LogFileOuts.ArgStr,
+                                       cl::ExecutableName.ArgStr,
+                                       cl::BuildShaderCache.ArgStr,
+                                       "o"};
```

Comment:
FWIW, I consider combining a significant style change with a semantic change like this a bit rude. It'd be nicer to have that split into separate commits. I can live with it here, though.

Reply:
Thanks for the suggestion. Do I understand correctly that you propose to first turn off clang-format here and add the extra options, and then remove the formatting exception and reformat the initializer in a subsequent commit?